config/features.yml

```diff
@@ -354,33 +354,6 @@ features:
   communication_preferences:
     actor_type: user
     description: Allow user to access backend communication_preferences API
-  covid_vaccine_registration:
```

Comment:
Reminder these features will need to be [removed in the rails console](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Removingold/unusedfeaturetoggles)